drivers/interface.go

```diff
@@ -182,44 +232,71 @@ func views(c ViewConstructor, schema string, whitelist, blacklist []string) ([]T
 
 	sort.Strings(names)
 
-	var views []Table
-	for _, name := range names {
-		t := Table{
-			IsView: true,
-			Name:   name,
-		}
+	ret := make([]Table, len(names))
+
+	limiter := newConcurrencyLimiter(concurrency)
+	wg := sync.WaitGroup{}
+	errs := make(chan error, len(names))
+	for i, name := range names {
+		limiter.get()
+		wg.Add(1)
```

Comment:
Very minor change, but I feel like maybe `limiter.get()` should come after `wg.Add(1)`.

While very unlikely, it feels like it is possible that `wg.Wait()` unblocks sometime between when one of the workers finish and another begins.

Reply:
This cannot happen here, because limiter.get is a blocking operation when there are no tickets available and wg.Wait() is called after all Add().
But you are right, wg.Add() should be called first just to be consistent with the cases where such situations are possible.